### DISCUSSION TITLE
How to update from 102 on a Pi

### DISCUSSION THREAD

**Discussion by Friggasvej** [AdguardTeam/AdGuardHome#3472] [Q&A]
Raspberry Pi 4 B (headless) running buster lite.
Currently running AdguardHome v102.

Anybody can help with cli command to update from v102 to newest betabranch?

Is auto update on the roadmap or is it only possible with snap?

**Comment by ainar-g** [comment]
(Before trying anything, backup your AGH data.)

Technically, something like:

```sh
curl -s -S -L https://raw.githubusercontent.com/AdguardTeam/AdGuardHome/master/scripts/install.sh | sh -s -- -c 'beta' -r -v
```

*could* work, but I don't think anyone has tested it with a release that old.  If that won't work, you'll probably have to manually download the latest beta version from the [releases page].

[releases page]: https://github.com/AdguardTeam/AdGuardHome/releases
